src/command-palette/plugin.tsx

```diff
@@ -0,0 +1,303 @@
+					sanitizedTitle
+				),
+				icon: activityPubIcon,
+				callback: ( { close }: { close: () => void } ) => {
+					document.location = `post.php?post=${ record.id }&action=edit`;
+					close();
+				},
+			};
+		} );
+	}, [ records ] );
+
+	return {
+		commands,
+		isLoading,
+	};
+};
+
+// User-specific commands (for actor and actor_blog modes).
+if ( actorMode === 'actor' || actorMode === 'actor_blog' ) {
+	// User Followers command.
+	registerCommand( {
+		name: 'activitypub/navigate-user-followers',
+		label: __( 'ActivityPub: View Your Followers', 'activitypub' ),
+		icon: activityPubIcon,
+		callback: ( { close } ) => {
+			document.location.href = 'users.php?page=activitypub-followers-list';
+			close();
+		},
+	} );
+
+	// User Following command (only if enabled).
+	if ( followingEnabled ) {
+		registerCommand( {
+			name: 'activitypub/navigate-user-following',
+			label: __( 'ActivityPub: View Who You Follow', 'activitypub' ),
+			icon: activityPubIcon,
+			callback: ( { close } ) => {
+				document.location.href = 'users.php?page=activitypub-following-list';
+				close();
+			},
+		} );
+	}
+
+	// User Extra Fields commands.
+	registerCommand( {
+		name: 'activitypub/navigate-extra-fields',
+		label: __( 'ActivityPub: View Extra Fields', 'activitypub' ),
+		icon: activityPubIcon,
+		callback: ( { close } ) => {
+			document.location.href = 'edit.php?post_type=ap_extrafield';
+			close();
+		},
+	} );
+
+	registerCommand( {
+		name: 'activitypub/add-extra-field',
+		label: __( 'ActivityPub: Add New Extra Field', 'activitypub' ),
+		icon: activityPubIcon,
+		callback: ( { close } ) => {
+			document.location.href = 'post-new.php?post_type=ap_extrafield';
+			close();
+		},
+	} );
+
+	// Dynamic command loader: Edit existing extra fields.
+	registerCommandLoader( {
+		name: 'activitypub/extra-fields-search',
+		hook: useExtraFieldsCommandLoader,
+	} );
+
+	// Blocked Actors command (user-specific).
+	registerCommand( {
+		name: 'activitypub/navigate-blocked-actors',
+		label: __( 'ActivityPub: View Blocked Actors', 'activitypub' ),
+		icon: activityPubIcon,
+		callback: ( { close } ) => {
+			document.location.href = 'users.php?page=activitypub-blocked-actors-list';
+			close();
+		},
+	} );
+}
+
+// Blog-related commands (for blog and actor_blog modes with manage_options capability).
+if ( canManageOptions && ( actorMode === 'blog' || actorMode === 'actor_blog' ) ) {
+	// Blog Followers command.
+	registerCommand( {
+		name: 'activitypub/navigate-blog-followers',
+		label: __( 'ActivityPub: View Blog Followers', 'activitypub' ),
+		icon: activityPubIcon,
+		callback: ( { close } ) => {
+			document.location.href = 'options-general.php?page=activitypub&tab=followers';
+			close();
+		},
+	} );
+
+	// Blog Following command (only if enabled).
+	if ( followingEnabled ) {
+		registerCommand( {
+			name: 'activitypub/navigate-blog-following',
+			label: __( 'ActivityPub: View Blog Following', 'activitypub' ),
```

Comment:
Let's use a similar wording for "View Who You Follow" and "View Blog Following", to make selections easier. Either use "View your Followings" or "View Who your Blog Follows" (<- or something that makes more sense in english 😉)

<img width="498" height="209" alt="Image" src="https://github.com/user-attachments/assets/5679d059-0022-460d-9ea0-9df8aefd1035" />

Reply:
Fixed in [c755f21](https://github.com/Automattic/wordpress-activitypub/pull/2315/commits/c755f21520bc71ae5595fcd343f1862aad095cf3).


<img width="418" height="401" alt="Screenshot 2025-10-16 at 1 21 57 PM" src="https://github.com/user-attachments/assets/ae932e77-8e93-4164-ad05-fdaa904536b0" />